app.js

```diff
@@ -24,143 +24,160 @@ SurveyItem.prototype.buildSurveyItem = function(conatainerId){
+];
 
-var itemObjects = [boots,chair,scissors,water_can,wine_glass,boots2,chair2,scissors2,water_can2];
 var itemsObjectsWorking;
+var itemsChosen = 0;
 
-function getFirstRandom(){
+function displaySingleRandom(elementIndex,locationID){
   var index = Math.floor(Math.random() * (itemsObjectsWorking.length));
-  if(document.getElementsByClassName('surveyImageBox')[0] && document.getElementsByClassName('surveyItemDesc')[0]){
-    var oldImage = document.getElementsByClassName('surveyImageBox')[0];
-    var oldItemDesc = document.getElementsByClassName('surveyItemDesc')[0];
+  if(document.getElementsByClassName('surveyImageBox')[elementIndex] && document.getElementsByClassName('surveyItemDesc')[elementIndex]){ //remove old elements if the exist
+    var oldImage = document.getElementsByClassName('surveyImageBox')[elementIndex];
+    var oldItemDesc = document.getElementsByClassName('surveyItemDesc')[elementIndex];
     oldImage.remove();
     oldItemDesc.remove();
   }
-  itemsObjectsWorking[index].buildSurveyItem('surveyFirstItem');
-  //itemsObjectsWorking.splice(index,1);
+  itemsObjectsWorking[index].buildSurveyItem(locationID); //build new element
+  itemsObjectsWorking.splice(index,1);
 }
 
-function getSecondRandom(){
-  var index = Math.floor(Math.random() * (itemsObjectsWorking.length));
-  if(document.getElementsByClassName('surveyImageBox')[1] && document.getElementsByClassName('surveyItemDesc')[1]){
-    var oldImage = document.getElementsByClassName('surveyImageBox')[1];
-    var oldItemDesc = document.getElementsByClassName('surveyItemDesc')[1];
-    oldImage.remove();
-    oldItemDesc.remove();
+function preventDuplicateDisplay(){
+  itemsObjectsWorking = itemObjects.slice();
+  var indexesToRemove = [];
+  for(var i = 0; i < itemsObjectsWorking.length; i++){
+    if(itemsObjectsWorking[i].usedInLastItemSet){
+      indexesToRemove.push(i);
+    }
   }
-  itemsObjectsWorking[index].buildSurveyItem('surveySecondItem');
-  //itemsObjectsWorking.splice(index,1);
+  for(var j = (indexesToRemove.length - 1); j > -1; j--){
+    itemsObjectsWorking.splice(indexesToRemove[j],1);
+  }
+  itemObjects.map(function(item) { //set all the SurveyItem.lastLoopUsed back to false to be ready for next set
+    item.usedInLastItemSet = false;
+  });
 }
 
-function getThirdRandom(){
-  var index = Math.floor(Math.random() * (itemsObjectsWorking.length));
-  if(document.getElementsByClassName('surveyImageBox')[2] && document.getElementsByClassName('surveyItemDesc')[2]){
-    var oldImage = document.getElementsByClassName('surveyImageBox')[2];
-    var oldItemDesc = document.getElementsByClassName('surveyItemDesc')[2];
-    oldImage.remove();
-    oldItemDesc.remove();
-  }
-  itemsObjectsWorking[index].buildSurveyItem('surveyThirdItem');
-  //itemsObjectsWorking.splice(index,1);
+function displayRandomItems(){
+  preventDuplicateDisplay();
+  displaySingleRandom(0,'surveyFirstItem');
```

Comment:
Almost always better to replace names with first, second, third, etc. with an array that represents the same concept.